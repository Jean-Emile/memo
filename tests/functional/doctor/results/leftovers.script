{
    "leftovers": [
        {
            "name": "foo",
            "reason": "should not be there",
            "sane": true,
            "warning": true
        },
        {
            "name": "linked_networks/bar",
            "reason": "json parse error",
            "sane": true,
            "warning": true
        }
    ],
    "networks": [
        {
            "linked": true,
            "name": "bob/aws_network",
            "sane": true,
            "warning": false
        },
        {
            "linked": true,
            "name": "bob/fs_network",
            "sane": true,
            "warning": false
        }
    ],
    "sane": true,
    "silos": [
        {
            "name": "aws_storage",
            "sane": true,
            "type": "S3",
            "warning": false
        },
        {
            "name": "fs_storage",
            "reason": "\"/tmp/\" is readable and writable",
            "sane": true,
            "type": "filesystem",
            "warning": false
        }
    ]
}
