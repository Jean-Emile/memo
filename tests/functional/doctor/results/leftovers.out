[OK] User
[OK] Silos
[OK] Networks
[WARNING] Leftovers:
  [WARNING] foo
  [WARNING] linked_networks/bar
